.gitignore

```diff
@@ -16,6 +24,14 @@ HasalServer.log.*
 stage/*
 docs/*
 
+# File types to ignore
+*.zip
+*.diff
+*.json
+*.flow
```

Comment:
We do have some .flow files. We use them to run with mitmproxy and those are precached flows.
https://github.com/Mozilla-TWQA/Hasal/tree/dev/flows